services/src/modules/config.ts

```diff
@@ -27,4 +26,10 @@ export const tmpPoliciesDir = envVar
   .asString();
 
 // Authentication
-export const authenticationConfig = envVar.get('AUTHENTICATION_CONFIGURATION').asJsonObject() as AuthenticationConfig;
+const defaultAuthenticationConfig: AuthenticationConfig = {
+  anonymous: {
+    paths: ['/metrics', '/.well-known/apollo/server-health', '/graphql'],
+  },
+};
+export const authenticationConfig =
+  (envVar.get('AUTHENTICATION_CONFIGURATION').asJsonObject() as AuthenticationConfig) ?? defaultAuthenticationConfig;
```

Comment:
I think this will also be fine with the `env-var` module default function?
`envVar.get('AUTHENTICATION_CONFIGURATION').default(defaultAuthenticationConfig).asJsonObject()`